sync3/dispatcher.go

```diff
@@ -19,10 +19,14 @@ var logger = zerolog.New(os.Stdout).With().Timestamp().Logger().Output(zerolog.C
 
 const DispatcherAllUsers = "-"
 
+// Receiver represents the callbacks that a Dispatcher may fire.
 type Receiver interface {
 	OnNewEvent(ctx context.Context, event *caches.EventData)
 	OnReceipt(ctx context.Context, receipt internal.Receipt)
 	OnEphemeralEvent(ctx context.Context, roomID string, ephEvent json.RawMessage)
+	// OnRegistered is called after a successful call to Dispatcher.Register. After
+	// this call, the receiver will be told about events whose NID is greater than
```

Comment:
s/events/PDUs

Reply:
(i.e. the receiver always gets told about all EDUs; EDUs have nothing like a NID to act as a sequence number)